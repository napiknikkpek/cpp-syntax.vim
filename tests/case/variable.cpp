int a = 3;
Foo t;
auto t = 3;
auto [a, b, c] = foo();
true ? a : b;
