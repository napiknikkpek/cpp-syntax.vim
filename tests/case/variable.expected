zzz.x...c.
ttt.x.
aaaa.x...c.
aaaa..x..x..x....fff...
cccc...x...x.
